examples/js/controls/EditorControls.js

```diff
@@ -7,7 +7,8 @@
 
 THREE.EditorControls = function ( object, domElement ) {
 
-	domElement = ( domElement !== undefined ) ? domElement : document;
+	if ( domElement === undefined ) console.error( 'THREE.EditorControls: The second paramter "domElement" is now mandatory. In most cases, "renderer.domElement" is best.' );
+	if ( domElement === document ) console.warn( 'THREE.EditorControls: "document" is no valid value for "domElement". Please use "renderer.domElement" instead.' );
```

Comment:
> "document" is no valid value for "domElement"

Maybe instead:

"document" is not a valid value for "domElement"

or

"document" cannot be used as the target "domElement".